tests/test_mmar_download.py

```diff
@@ -85,18 +86,26 @@ class TestMMMARDownload(unittest.TestCase):
     @skip_if_quick
     @SkipIfBeforePyTorchVersion((1, 6))
     def test_download(self, idx):
-        download_mmar(idx)
-        download_mmar(idx, progress=False)  # repeated to check caching
-        with tempfile.TemporaryDirectory() as tmp_dir:
-            download_mmar(idx, mmar_dir=tmp_dir, progress=False)
-            download_mmar(idx, mmar_dir=tmp_dir, progress=False)  # repeated to check caching
-            self.assertTrue(os.path.exists(os.path.join(tmp_dir, idx)))
+        try:
+            download_mmar(idx)
+            download_mmar(idx, progress=False)  # repeated to check caching
+            with tempfile.TemporaryDirectory() as tmp_dir:
+                download_mmar(idx, mmar_dir=tmp_dir, progress=False)
+                download_mmar(idx, mmar_dir=tmp_dir, progress=False)  # repeated to check caching
+                self.assertTrue(os.path.exists(os.path.join(tmp_dir, idx)))
+        except (ContentTooShortError, HTTPError, RuntimeError) as e:
+            print(str(e))
```

Comment:
Could you please add a basic test for the error message? Just lie:
https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_download_and_extract.py#L36

Thanks.